Passport.js

```diff
@@ -0,0 +1,58 @@
+import passport from 'passport'
+import passportJWT from 'passport-jwt'
+import { Pool } from 'pg';
+
+import DbQuery from './server/model/';
+import { comparePwd } from './server/helpers/pwdHash';
+
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJWT = passportJWT.ExtractJwt;
+const LocalStrategy = require('passport-local').Strategy;
+
+passport.use(new JWTStrategy({
+	jwtFromRequest: ExtractJWT.fromHeader('authorization'),
+	secretOrKey: 'secret'	
+}, async (payload, done) => {
+	try{
+		const pool = new Pool();
+		const user = await DbQuery.getUserWithEmail(payload.email)
+		let dbRes = await pool.query(user);
+		await pool.end();
+		console.log(dbRes.rows[0])
```

Comment:
@olamilekan000 Nice work, however, try not to push `console.log` to Github.

Reply:
Noted